Makefile.dep

```diff
@@ -416,12 +416,20 @@ ifneq (,$(filter posix_sockets,$(USEMODULE)))
   USEMODULE += vfs
 endif
 
+ifneq (,$(filter shell,$(USEMODULE)))
+  USEMODULE += uart_stdio_rx
+endif
+
 ifneq (,$(filter rtt_stdio,$(USEMODULE)))
   USEMODULE += xtimer
 endif
 
-ifneq (,$(filter uart_stdio,$(USEMODULE)))
+ifneq (,$(filter uart_stdio_rx,$(USEMODULE)))
   USEMODULE += isrpipe
+  USEMODULE += uart_stdio
```

Comment:
Might be confusing why there is uart_stdio and uart_stdio_rx - is renaming uart_stdio to uart_stdio_tx useful/helpful/required? Or should we add uart_stdio_tx as a pseudo module just to make it clear but also have the opportunity to have both?